packages/platforms/accton/x86-64/x86-64-accton-wedge100bf-32x/platform-config/r0/src/python/x86_64_accton_wedge100bf_32x_r0/__init__.py

```diff
@@ -0,0 +1,23 @@
+from onl.platform.base import *
+from onl.platform.accton import *
+
+class OnlPlatform_x86_64_accton_wedge100bf_32x_r0(OnlPlatformAccton,
+                                                OnlPlatformPortConfig_32x100):
+    MODEL="Wedge-100bf-32X"
+    PLATFORM="x86-64-accton-wedge100bf-32x-r0"
+    SYS_OBJECT_ID=".100.32.1"
```

Comment:
The x86-64-accton-wedge100-32x-r0 already uses this SYS_OBJECT_ID. 
These must be unique per platform. Please I suggest .100.32.2.